android/pytorch_android/src/androidTest/java/org/pytorch/PytorchInstrumentedTests.java

```diff
@@ -228,16 +231,16 @@ public void testTensorMethods() {
       floats[i] = i / 1000.f;
     }
 
-    Tensor tensorBytes = Tensor.newByteTensor(dims, bytes);
-    assertTrue(tensorBytes.isByteTensor());
+    Tensor tensorBytes = Tensor.newTensor(shape, bytes);
+    assertTrue(tensorBytes.dtype() == Tensor.DTYPE_BYTE);
```

Comment:
we don't actually have a Byte dtype in the python frontend (because numpy defines it differently).  It's probably better to use `DTYPE_UINT8`.